src/start_menu.c

```diff
@@ -0,0 +1,1556 @@
+        break;
+    case START_MENU_STATE_10:
+        if (sub_020505C8(fieldSystem)) {
+            ov01_021E636C(1);
+            startMenu->state = START_MENU_STATE_11;
+        }
+        break;
+    case START_MENU_STATE_11:
+        if (IsPaletteFadeFinished()) {
+            sub_0203C69C(startMenu, fieldSystem);
+            FreeToHeap(startMenu);
+            MapObjectManager_UnpauseAllMovement(fieldSystem->mapObjectManager);
+            return TRUE;
+        }
+        break;
+    case START_MENU_STATE_12:
+        if (sub_020505C8(fieldSystem)) {
+            MapObjectManager_PauseAllMovement(fieldSystem->mapObjectManager);
+            ov01_021E636C(1);
+            startMenu->state = START_MENU_STATE_13;
+        }
+        break;
+    case START_MENU_STATE_13:
+        if (IsPaletteFadeFinished()) {
+            TaskManager_Jump(taskManager, startMenu->atexit_TaskFunc, startMenu->atexit_TaskEnv);
+            FreeToHeap(startMenu);
+        }
+        break;
+    case START_MENU_STATE_14:
+        startMenu->atexit_TaskFunc(taskManager);
+        break;
+    case START_MENU_STATE_18:
+        FreeToHeap(startMenu);
+        MapObjectManager_UnpauseAllMovement(fieldSystem->mapObjectManager);
+        return TRUE;
+    case START_MENU_STATE_CLOSE:
+        sub_0203C69C(startMenu, fieldSystem);
+        sub_0203C38C(startMenu, fieldSystem);
+        fieldSystem->unkD2_0 = 2;
+        ScheduleBgTilemapBufferTransfer(fieldSystem->bgConfig, GF_BG_LYR_MAIN_3);
+        FreeToHeap(startMenu);
+        MapObjectManager_UnpauseAllMovement(fieldSystem->mapObjectManager);
+        return TRUE;
+    case START_MENU_STATE_RETURN_WAIT_FADE:
+        if (IsPaletteFadeFinished()) {
+            startMenu->state = START_MENU_STATE_HANDLE_INPUT;
+        }
+        break;
+    case START_MENU_STATE_19:
+        sub_0203D2EC(taskManager);
+        break;
+    case START_MENU_STATE_20:
+        sub_0203D304(taskManager);
+        break;
+    }
+    if (startMenu->cursorActive) {
+        sub_0202457C(startMenu->spriteList);
+    }
+    return FALSE;
+}
+
+static void Task_StartMenu_DrawCursor(TaskManager *taskManager) {
+    FieldSystem *fieldSystem = TaskManager_GetFieldSystem(taskManager);
+    StartMenuTaskData *startMenu = (StartMenuTaskData *)TaskManager_GetEnvironment(taskManager);
+
+    u32 numActiveButtons = StartMenu_BuildActionLists(startMenu, startMenu->insertionOrder, startMenu->selectionToAction);
+    startMenu->numActiveButtons = numActiveButtons;
+    startMenu->lastButtonSelected = 0;
+    for (int i = 0; i < numActiveButtons; ++i) {
+        if (fieldSystem->unk90 == startMenu->insertionOrder[i]) {
+            startMenu->lastButtonSelected = i;
+        }
+    }
+    startMenu->cursorActive = TRUE;
+    GfGfxLoader_LoadCharData(NARC_a_0_1_4, 12, fieldSystem->bgConfig, GF_BG_LYR_MAIN_3, 0, 0, TRUE, HEAP_ID_FIELD);
+    GfGfxLoader_GXLoadPal(NARC_a_0_1_4, 15, GF_PAL_LOCATION_MAIN_BG, (enum GFPalSlotOffset)0x1C0, 0x20, HEAP_ID_FIELD);
```

Comment:
0x1c0 is an offset you can use (the enum has it)

Reply:
as previously stated, fuck this enum in particular